src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -16,33 +17,61 @@ export interface AdminVenuesProps {
 }
 
 export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
+  const [
+    currentSortingOption,
+    setCurrentSortingOption,
+  ] = useState<VenueSortingOptions>(VenueSortingOptions.az);
+
+  const sortedVenues = useMemo(() => {
+    return sortVenues(venues, currentSortingOption) ?? [];
+  }, [currentSortingOption, venues]);
```

Comment:
Just a boilerplate trim

```suggestion
  const sortedVenues = useMemo(
    () => sortVenues(venues, currentSortingOption) ?? [],
    [currentSortingOption, venues]
  );
```